docs/docs/environment-variables.md

```diff
@@ -106,7 +106,7 @@ render() {
 }
 ```
 
-`API_KEY` will be available to your site (Server-side) as `process.env.API_KEY`. If you commit your `.env.*` file containing `API_KEY` to source control it would also be available on the client-side. However we **strongly** advise against that! You should prefix your variable with `GATSBY_` (as shown above) instead and Gatsby automatically makes it available in the browser context.
+In Node, `API_KEY` will be available to your site (Server-side) as `process.env.API_KEY`. You can use `dotenv` to commit your `.env.*` file containing `API_KEY` in order to make it available client-side, but we recommend against checking these files into source control. By prefixing your variable with `GATSBY_` (as shown above), Gatsby automatically embeds it as process.env.GATSBY_* in compiled JS making it available in the browser context.
```

Comment:
```suggestion
In Node, your site has access to `API_KEY` (Server-side) using the identifier `process.env.API_KEY`. To access it client-side you can use a `.env.*` file containing `API_KEY`. However, we **strongly** advise against checking these files into source control as its a security issue to expose the API key. As a more secure alternative, you can prefix your variable with `GATSBY_` (as shown above). With this prefix, Gatsby automatically embeds the variable as process.env.GATSBY_* in compiled JS making it available in the browser context without exposing it elsewhere.
```

Reply:
Hi @laurieontech , thank you for the suggestions! I incorporated them into the latest commit. Please let me know if there are any other changes you'd recommend!